File: Java Old Code/Class9/src/Armstrongnumber.java
/*Write a C program to check whether a number is Armstrong number or not.*/
import java.io.*;
public class Armstrongnumber
{
    public static void main ( String args [])throws IOException
    { 
        DataInputStream dis = new DataInputStream(System.in);
        System.out.println("ENTER A NUMBER");
       int a = Integer.parseInt(dis.readLine());
       int s = 0;
       int r = a ;
       while( a  > 0)
       {
        int b = a % 10;
        s= s + (b*b*b); 
       
       a = a /10;
      }
    if ( s==r)
    {
        System.out.println(r+" IS A ARMSTRONG NUMBER");
    }
    else
    {
        System.out.println(r+" IS NOT A ARMSTRONG NUMBER");
    }
    }
}

File: C_Programs/ArmstrongNumber.c
#include <stdio.h>
int main()
{
    for (int i = 1; i <= 500; i++)
    {
        int temp = i;
        int result = 0;

        while (temp != 0)
        {
	    int x = temp%10;
            result += x*x*x;
            temp /= 10;
        }

        if (result == i)
            printf("%d is an Armstrong Number\n", i);
    }

    return 0;
}
